openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/TableRepository.java

```diff
@@ -1084,7 +1084,7 @@ public void entitySpecificUpdate() throws IOException {
       DatabaseUtil.validateColumns(updatedTable);
       recordChange("tableType", origTable.getTableType(), updatedTable.getTableType());
       updateConstraints(origTable, updatedTable);
-      updateColumns("columns", origTable.getColumns(), updated.getColumns(), EntityUtil.columnMatch);
+      updateColumns("columns", origTable.getColumns(), updated.getColumns(), EntityUtil.columnNameMatch);
```

Comment:
`EntityUtil.columnNameMatch` does a case insensitive comparison of column names, It should be a case sensitive comparison because in database like snowflake it's possible to have two columns with same name but different case